### DISCUSSION TITLE
further on eWidgets with Sprites

### DISCUSSION THREAD

**Discussion by peteDDD** [Bodmer/TFT_eSPI#2119] [Q&A - Sprites]
Using the GraphWidget and TraceWidget, is it possible to put just the Trace in a Sprite while leaving the Graph or does the Trace always have to be a daughter of the Graph?  (Various attempts on my part fail...)

**Comment by Bodmer** [comment]
I am not sure what "while leaving the Graph" means. But maybe this is the answer (if not please elaborate):

The graph can be chosen to either be associated with the TFT or associated with a sprite. The trace must be linked to the graph (to get the scale factors, graph size etc etc) and thus is associated with your previous choice.

**Reply by peteDDD** [reply]
Thanks for the reply.  That is how I understood things.  What I was hoping for is to treat the trace like one treats a needle on a gauge (making it a Sprite, and treat the chart background (ticks, etc) like one treats the background on a gauge.  But I understand how they are linked.
